src/util/algolia.js

```diff
@@ -43,10 +43,15 @@ const pageQuery = `{
       
   const queries = [
     {
-      query: pageQuery,
+      query: generatePageQuery(),
       transformer: ({ data }) => flatten(data.articles.edges),
       indexName: `learning_center`,
     },
+    {
+      query: generatePageQuery("es"),
+      transformer: ({ data }) => flatten(data.articles.edges),
+      indexName: `learning_center_es`,
```

Comment:
hey @toolness! Here is my implementation of Algolia's search index generation for different locales. All I did was abstract out the graphql query that generates content for the index, and include a "locale" variable so we can define which language to download. Then, I just called that generate function twice, once for the default locale and again for Spanish ("es"). Thus, any build of our site will create or update two disjoint search indexes for each locale we currently support.

Reply:
Ah nice, that sounds like a great solution!

Reply:
yay!